### ISSUE TITLE
Is FAILURES_ALLOWED working for anyone?

### ISSUE THREAD

**Issue by manolomanila** [Noctem/Monocle#248]
I know a nearby spawn point doesn't spawn mons anymore, but it still has despawn info.

This might be related to issue #247.

**Comment by Jabsu** [comment]
Well, the configuration itself works and the log is filled with "consecutive failures" regarding spawn points, but I guess you didn't mean that. What you probably meant was, that rather than treating those removed spawn points as unknown, the workers should forget about them. So in that sense, it doesn't work, and having a lot of unknown spawns make the workers restless.

Don't know if this is related, but my Elevation API quota has been getting exceeded every day after these spawn failures surfaced. Luckily I live in an area where there aren't huge differences in altitudes, so the random numbers between my ALT_RANGE values do their job. It's still a bit risky, though.

**Comment by manolomanila** [comment]
Yeah, I'm getting these failures in the log file as well. I guess I didn't (or still don't) understand well how it's supposed to work; I thought it would remove the spawn point despawn time to consider it an unkonwn (so you would see it as red) but what I understand from your comment and after reading the conf file again is that it may treat it as an unknown without removing the despawn value.

Still, what you suggested makes a lot of sense. These spawns should be deleted rather than treated as unknown. Or deleted after a certain amount of time, if the workers can't find any mon spawning.

I haven't exceeded my elevation quota but I've been getting some altitude errors. I'm going to turn on ALT_RANGE and see if I get less of both, altitude errors and consecutive failures.

**Comment by manolomanila** [comment]
Checked on the db file a couple of spawnpoints that no longer exist to see if it was a web.py bug, and they have 0 failures and despawn_time data, so there is definitely a bug regarding the db file or the unkonwn spawns.

Also, not a single error or failure occurred since I turned on ALT_RANGE. @Jabsu could it be that your range is too wide? I set up mine to go from 0 to 70 as 70m is the user area radius.

**Comment by Jabsu** [comment]
ALT_RANGE is always 'on', it just uses default values (300, 400) if it's commented. As a reminder, ALT_RANGE is only used when Google Maps Elevation API query fails. When that happens, Monocle will pick a random number between that range. The range should represent the real minimum and maximum altitudes of your scanning area (e.g. my range is only 39-60 as there's not much elevation in my city). 

Anyway, there's a lot of spawn related fixes incoming, as you can see on the WIP branch. Can't wait! :)
